slacker.py

```diff
@@ -18,6 +18,26 @@ def __init__(self, slack, token):
         self.api_calls = 0
         self.api_wait = 0
 
+    def conditional_set_topic(self, cid, topic):
+        """
+        Set the topic for cid it it's not already the expected topic
+        """
+        current_topic = self.get_topic(cid)
+        if current_topic != topic:
+            self.set_topic(cid, topic)
+
+    def get_topic(self, cid):
+        response = self.api_call("conversations.info?channel={}".format(cid))
+        return response .get("channel", {}).get("topic", {}).get("value")
```

Comment:
Is that a space between response and .get?

Reply:
Yes.  Fixed.